getting-started/kubernetes/self-managed-onprem/onpremises.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Install Calico networking and network policy for on-premises deployments
-description: Install Calico networking and network policy for on-premises deployments.
+description: Determine your datastore and use Calico manifest to customize the resources on each node in the cluster. 
```

Comment:
Suggest revert to the original text.  (Determining your datastore and using Calico manifests are steps in the how-to guide; they aren't a description of the how-to guide's goal.)

The Customize Manifest description is worth updating though (getting-started/kubernetes/installation/config-options.md):
Customize manifests prior to installing Calico (recommended) -> Optionally customize manifests prior to installing Calico.

Reply:
Fixed